doc/source/whatsnew/v1.5.0.rst

```diff
@@ -109,6 +109,32 @@ apply converter methods, and parse dates (:issue:`43567`).
     df
     df.dtypes
 
+read_html now supports ``extract_hrefs``
```

Comment:
Its enough to add a one line in the whats new, something like:

- read_html adds a keyword argument `extract_hrefs` to parse just the hyperlinks.

Then in the documentation for the function you can give a minimalist example of the keyword use

Reply:
OK, will do - I just copied the `read_excel` section above and adapted it for my changes.

Reply:
> Then in the documentation for the function you can give a minimalist example of the keyword use

Where would I put it?

Reply:
Also, why shouldn't it be similar to the whatsnew entry for `read_excel`? I assumed that, since my changes are in a similar vein to the changes in the `read_excel` PR, I could copy and adapt the structure of it for my changes.

Reply:
I don't believe they are similar in vein. The changes in `read_xml` are fundamentally significant and, apparently breaking and apply in all cases.

Your change applies only in the event a user has hrefs to parse and chooses to optionally parse the URL which is probably quite rare, and in the default case it has no effect.

> Where would I put it?

Add to the `Examples` section of the function documentation for `read_html`. This is a better place anyway since the release notes are not persistent documentation, whereas the method documentation is.

Note, that in my opinion the section for `read_xml` in the what's new notes is also probably too extensive. It might be trimmed down or edited before the release.

Reply:
@attack68 Is it OK now?